Modules/Microsoft365DSC/Modules/M365DSCReport.psm1

```diff
@@ -516,7 +524,7 @@ function Compare-M365DSCConfigurations
                 # The resource instance exists in both the source and the destination. Compare each property;
                 foreach ($propertyName in $sourceResource.Keys)
                 {
-                    if ($propertyName -notin @("ResourceName", "ResourceId", "Credential", "CertificatePath", "CertificatePassword", "TenantId", "ApplicationId", "CertificateThumbprint", "ApplicationSecret"))
+                    if ($propertyName -notin @('ResourceName', 'ResourceId', 'Credential', 'CertificatePath', 'CertificatePassword', 'TenantId', 'ApplicationId', 'CertificateThumbprint', 'ApplicationSecret'))
```

Comment:
Should we add 'ManagedIdentity' to this list as well?

Reply:
Done